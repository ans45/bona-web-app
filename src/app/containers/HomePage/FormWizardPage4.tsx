import Box from '@material-ui/core/Box';
import { FormWizardPage } from 'app/components/FormWizard/FormWizardPage';
import { FormWizardPageTitle } from 'app/components/FormWizardPageTitle';
import React from 'react';
import { Checkboxes } from 'mui-rff';

export const FormWizardPage4 = () => (
  <FormWizardPage validate={() => undefined} submitButtonText="Submit">
    <Box mb={4}>
      <FormWizardPageTitle>Center aligned question?</FormWizardPageTitle>
      <Checkboxes
        name="checkboxInput"
        data={[
          { label: 'Short phrase', value: 'item1' },
          { label: 'Really long sentence', value: 'item2' },
          { label: 'Notice the left alignment', value: 'item3' },
          { label: 'Default is nothing selected', value: 'item4' },
        ]}
      />
    </Box>
  </FormWizardPage>
);
