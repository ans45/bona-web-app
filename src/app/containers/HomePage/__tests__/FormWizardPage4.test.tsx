import React from 'react';
import { render } from '@testing-library/react';
import { FormWizardPage4 } from '../FormWizardPage4';
import { Form } from 'react-final-form';

describe('<FormWizardPage4  />', () => {
  it('should match snapshot', () => {
    const { container } = render(
      <Form onSubmit={() => undefined}>{() => <FormWizardPage4 />}</Form>,
    );
    expect(container.firstChild).toMatchSnapshot();
  });
});
