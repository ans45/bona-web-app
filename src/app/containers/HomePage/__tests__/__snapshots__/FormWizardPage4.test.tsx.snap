// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FormWizardPage4  /> should match snapshot 1`] = `
<div
  class="MuiBox-root MuiBox-root-1"
>
  <div
    class="MuiBox-root MuiBox-root-4"
  >
    <h2
      class="MuiTypography-root makeStyles-root-2 makeStyles-root-3 MuiTypography-h5"
    >
      Center aligned question?
    </h2>
  </div>
  <div
    class="MuiFormControl-root"
  >
    <div
      class="MuiFormGroup-root"
    >
      <label
        class="MuiFormControlLabel-root"
      >
        <span
          aria-disabled="false"
          class="MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-5 MuiCheckbox-root MuiCheckbox-colorSecondary MuiIconButton-colorSecondary"
        >
          <span
            class="MuiIconButton-label"
          >
            <input
              class="PrivateSwitchBase-input-8"
              data-indeterminate="false"
              name="checkboxInput"
              type="checkbox"
              value="item1"
            />
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M19 5v14H5V5h14m0-2H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z"
              />
            </svg>
          </span>
          <span
            class="MuiTouchRipple-root"
          />
        </span>
        <span
          class="MuiTypography-root MuiFormControlLabel-label MuiTypography-body1"
        >
          Short phrase
        </span>
      </label>
      <label
        class="MuiFormControlLabel-root"
      >
        <span
          aria-disabled="false"
          class="MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-5 MuiCheckbox-root MuiCheckbox-colorSecondary MuiIconButton-colorSecondary"
        >
          <span
            class="MuiIconButton-label"
          >
            <input
              class="PrivateSwitchBase-input-8"
              data-indeterminate="false"
              name="checkboxInput"
              type="checkbox"
              value="item2"
            />
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M19 5v14H5V5h14m0-2H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z"
              />
            </svg>
          </span>
          <span
            class="MuiTouchRipple-root"
          />
        </span>
        <span
          class="MuiTypography-root MuiFormControlLabel-label MuiTypography-body1"
        >
          Really long sentence
        </span>
      </label>
      <label
        class="MuiFormControlLabel-root"
      >
        <span
          aria-disabled="false"
          class="MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-5 MuiCheckbox-root MuiCheckbox-colorSecondary MuiIconButton-colorSecondary"
        >
          <span
            class="MuiIconButton-label"
          >
            <input
              class="PrivateSwitchBase-input-8"
              data-indeterminate="false"
              name="checkboxInput"
              type="checkbox"
              value="item3"
            />
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M19 5v14H5V5h14m0-2H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z"
              />
            </svg>
          </span>
          <span
            class="MuiTouchRipple-root"
          />
        </span>
        <span
          class="MuiTypography-root MuiFormControlLabel-label MuiTypography-body1"
        >
          Notice the left alignment
        </span>
      </label>
      <label
        class="MuiFormControlLabel-root"
      >
        <span
          aria-disabled="false"
          class="MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-5 MuiCheckbox-root MuiCheckbox-colorSecondary MuiIconButton-colorSecondary"
        >
          <span
            class="MuiIconButton-label"
          >
            <input
              class="PrivateSwitchBase-input-8"
              data-indeterminate="false"
              name="checkboxInput"
              type="checkbox"
              value="item4"
            />
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M19 5v14H5V5h14m0-2H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z"
              />
            </svg>
          </span>
          <span
            class="MuiTouchRipple-root"
          />
        </span>
        <span
          class="MuiTypography-root MuiFormControlLabel-label MuiTypography-body1"
        >
          Default is nothing selected
        </span>
      </label>
    </div>
  </div>
</div>
`;
